submariner-operator/issue-359-subctl-info-status.md

```diff
@@ -0,0 +1,116 @@
+#Issue 359: Add more information to subctl info cmd
+
+## Summary
+Current cmd:  
+* subctl info  
+    Discovered network details:  
+         Network plugin:  weave-net  
+         Service CIDRs: [100.92.0.0/16]  
+         Cluster CIDRs: [10.242.0.0/16]
+
+
+## Motivation
+To supply more information on current deployment status
+
+## Proposal
+* _subctl status network_  
```

Comment:
I think we should follow the verb+noun model followed in k8s cli. Verb in this case would be `show`. I was thinking something on lines of:
`subctl show status gateway`
`subctl show status network`
`subctl show config network`
`subctl show status deployment` etc.

We'll add to it over time, but think would be good to first have base foundation that can be easily extended for anything and everything in future.

Reply:
I think that's too many things on one line to do something. I believet it's better to stick to max 1 subcommand otherwise it becomes cumbersome to type

Reply:
I agree, we should keep the cmd simple and easy to use